TOC.md

```diff
@@ -175,7 +175,12 @@
       - [表库过滤](tools/lightning/filter.md)
       - [CSV 支持](tools/lightning/csv.md)
       - [监控告警](tools/lightning/monitor.md)
-    - [TiDB-Binlog](tools/tidb-binlog-cluster.md)
+    + TiDB-Binlog
+      - [概述](tools/binlog/overview.md)
+      - [部署使用](tools/binlog/deploy.md)
```

Comment:
@GregoryIan @WangXiangUSTC Since we have 3 versions of TiDB-Binlog, I think we might need to differentiate them in the file name. @ericsyh

Reply:
3 versions of TiDB-Binlog:
1. Latest version: TiDB-Binlog Cluster
2. Previous version: TiDB-Binlog Local
3. Kafka version: TiDB-Binlog Kafka

update the previous version file name in https://github.com/ericsyh/docs-cn/commit/d532c6e6675c82fb1bfc83b93d394e8bf2d8e158